frontend/src/components/kl-delete-all.js

```diff
@@ -1,8 +1,8 @@
-import api from '../../../config/api.json';
+import API from 'api.json';
```

Comment:
Make an alias directly to the api.json then? Or have two config dirs - one in backend, another in domain.